packages/clipboard/src/index.ts

```diff
@@ -44,6 +44,31 @@ const createClipboard = (): [
   return [write, read, { newItem }];
 };
 
+/**
+ * Creates a new reactive primitive for managing the clipboard.
+ *
+ * @return write - A method to write to the clipboard
+ * @return read - Read from the clipboard
+ * @return helpers - helper functions:
```

Comment:
old

Reply:
Yeah I haven't adjusted ANY of these yet. Just focusing on the API first.